WalletWasabi.Fluent/Styles/NotificationCard.axaml

```diff
@@ -1,28 +1,6 @@
 <Styles xmlns="https://github.com/avaloniaui"
-        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-        xmlns:c="using:WalletWasabi.Fluent.Controls">
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
   <Style Selector="NotificationCard">
-    <Setter Property="Template">
-      <ControlTemplate>
-        <LayoutTransformControl Name="PART_LayoutTransformControl" UseRenderTransform="True">
-          <Border CornerRadius="{TemplateBinding CornerRadius}" BoxShadow="0 6 8 0 #4F000000" Margin="5 5 5 10">
-            <Border Background="Transparent"
-                    BorderBrush="{DynamicResource GlassEdgeColor}"
-                    BorderThickness="{TemplateBinding BorderThickness}"
-                    CornerRadius="{TemplateBinding CornerRadius}"
-                    ClipToBounds="True">
-              <Panel>
-                <c:BlurBehind BlurRadius="10,10"/>
-                <Panel Background="{DynamicResource TileRegionColor}" Opacity="0.35" />
-                <DockPanel>
-                  <Panel x:Name="PART_HeaderBar" Height="4" DockPanel.Dock="Top" />
-                  <ContentControl Name="PART_Content" Content="{TemplateBinding Content}" />
-                </DockPanel>
-              </Panel>
-            </Border>
-          </Border>
-        </LayoutTransformControl>
-      </ControlTemplate>
-    </Setter>
```

Comment:
uhm no this shouldn't have been deleted?

Reply:
> SearchBarDropdown.axaml:
> 
> ```
>     <!-- TODO: Investigate why this is needed. Removing it will render monospaced font in Flyout -->
>     <Style Selector="TextBlock">
>       <Setter Property="FontFamily" Value="Arial" />
>     </Style>
> ```
> 
> Just noticed this.
> 
> It is because the Flyout is modified here: https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi.Fluent/Styles/FlyoutPresenter.axaml#L36
> 
> Please fix it in a separate PR.

PR created https://github.com/zkSNACKs/WalletWasabi/pull/7867

Reply:
upon review with @SuperJMN this change is okay for me 👍